altair/examples/stripplot.py

```diff
@@ -0,0 +1,39 @@
+"""
+Stripplot
+-----------------
+This example shows how to make a Stripplot.
+"""
+# category: scatter plots
+import altair as alt
+from vega_datasets import data
+
+source = data.movies()
```

Comment:
Since the movies data is not local, please use ``data.movies.url`` instead (this is to keep the test suite from requiring http access on the Python side)

Reply:
fixed